docs/docusaurus/docs/reference/learn/data_quality_use_cases/missingness.md

```diff
@@ -1,8 +1,153 @@
+|----------|------------------------|--------------------|-----------------|---------------------|--------|
+| domestic | 244084670977           | Jaxson Duke        | 9143.40         | 2024-05-01 01:12    | NULL   |
+| NULL     | 123456789012           | Jane Smith         | 5000.00         | NULL                | NULL   |
+
+This dataset includes potential for missing data in `type` and `transfer_date` columns.
+
+You can [access this dataset](https://raw.githubusercontent.com/great-expectations/great_expectations/develop/tests/test_sets/learn_data_quality_use_cases/missingness.csv) from the `great_expectations` GitHub repo in order to reproduce the code recipes provided in this article.
+
+
+## Key missingness Expectations
+
+GX provides a suite of missingness-focused Expectations to manage missing data in your datasets. These Expectations can be added to an Expectation Suite via the GX Cloud UI or using the GX Core Python library.
+
+
+:::[TODO]
+> Add animated GIF demonstrating how to create Expectations using the GX Cloud UI.
+:::
+
+### Expect Column Values To Be Null
+
+Ensures that values within a column are `NULL`.
+
+**Use Case**: Handle columns that should typically be populated but might be left null due to specific conditions.
+
+```python title="" name="docs/docusaurus/docs/reference/learn/data_quality_use_cases/missingness_resources/missingness_expectations.py ExpectColumnValuesToBeNull"
+gxe.ExpectColumnValuesToBeNull('errors')
+```
+
+<sup>View `ExpectColumnValuesToBeNull` in the [Expectation Gallery](https://greatexpectations.io/expectations/expect_column_values_to_be_null).</sup>
+
+### Expect Column Values To Not Be Null
+
+Ensures that values within a specific column are not `NULL`.
+
+**Use Case**: Ensure critical columns are consistently populated, such as `transfer_amount`.
+
+```python title="Python" name="docs/docusaurus/docs/reference/learn/data_quality_use_cases/missingness_resources/missing_expectations.py ExpectColumnValuesToNotBeNull"
+gxe.ExpectColumnValuesToNotBeNull('transfer_amount')
+```
+
+<sup>View `ExpectColumnValuesToNotBeNull` in the [Expectation Gallery](https://greatexpectations.io/expectations/expect_column_values_to_not_be_null).</sup>
+
+<br/>
+<br/>
+
+:::tip[GX tips for missingness Expectations]
+- Set different tolerance levels for `ExpectColumnValuesToBeNull` and `ExpectColumnValuesToNotBeNull`. This allows for nuanced data quality checks that align with your specific data patterns and business needs. For example, you might allow up to 20% nulls in a 'customer_feedback' column, but want to be alerted if more than 1% of values are unexpectedly non-null in an 'errors' column.
+- Use these Expectations to track data as it progresses through your pipeline. Apply `ExpectColumnValuesToBeNull` to fields expected to be empty in early stages, and `ExpectColumnValuesToNotBeNull` to those same fields in later stages.
+:::
+
+## Managing types of missing data
```

Comment:
I'm still thinking through this section, but let me provide my initial comments/questions.
* Would you consider this section to be the equivalent of the _Examples and scenarios_ section in the schema doc?
* I'd assumed that we would have longer code recipes in this section, like the schema doc, but I quite like how you've broken it down by types of missing data. I think it might be a more relevant way to communicate how GX can help address missingness. Let me think a bit more about code recipe content.
* Do you think these sections can be re-crafted to follow the Context/Goal/Insight pattern that we used for the Schema doc?

Reply:
1. I renamed the section to match Schema's. 
2. 👌 
3. I gave it a try, but terseness of each example makes subsections less effective here compared to schema's longer scenarios